lib/Doctrine/DBAL/Configuration.php

```diff
@@ -79,6 +80,12 @@ public function setResultCacheImpl(Cache $cacheImpl)
      */
     public function setFilterSchemaAssetsExpression($filterExpression)
     {
+        Deprecation::trigger(
+            'doctrine/dbal',
+            'https://github.com/doctrine/dbal/pull/3316',
+            'Configuration::setFilterSchemaAssetsExpression() is deprecated, use setSchemaAssetsFilter() instead.'
```

Comment:
Would it make sense to use `('%s()', __METHOD__)` instead of the actual method name? This may simplify the process of composing such messages but on the other hand, will make the code less readable. I'm fine with either approach.

Reply:
Yes, my thought was that decomposing the messages for constant strings is a bit annoying and that the FQCN don't add value why i did it this way, not sure about it though. I think it would help if its readable for humans, especially since the alternative is written out anyways.